appveyor.yml

```diff
@@ -29,6 +29,8 @@ build_script:
   - go test -c ./cri-containerd/ -tags functional
   - go test -c ./functional/ -tags functional
   - go test -c ./runhcs/ -tags functional
+  - cd ./cri-containerd/helpers
+  - go build -o sample-logging-driver.exe log.go
```

Comment:
Is there a specific reason we build the binary in /cri-containerd/helpers? Could we not do 

```
go build -o sample-logging-driver.exe ./cri-containerd/helpers/log.go
```

And then we could avoid adding the part about `TEST_BINARY_ROOT` 
What do you think?

Reply:
not really, just wanted to have the executable closer to the file. the `TEST_BINARY_ROOT` is more of a convenience to enable supplying an arbitrary location and the logic wasn't motivated by this.